HymnsTests/Tags/TagsStoreRealmImplSpec.swift

```diff
@@ -0,0 +1,57 @@
+import Combine
+import Quick
+import Mockingbird
+import Nimble
+import RealmSwift
+@testable import Hymns
+
+class TagStoreRealmImplSpec: QuickSpec {
+    override func spec() {
+        describe("using an in-memory realm") {
+            var inMemoryRealm: Realm!
+            var target: TagStoreRealmImpl!
+            beforeEach {
+                // Don't worry about force_try in tests.
+                // swiftlint:disable:next force_try
+                inMemoryRealm = try! Realm(configuration: Realm.Configuration(inMemoryIdentifier: "TagStoreRealmImplSpec"))
+                target = TagStoreRealmImpl(realm: inMemoryRealm)
+            }
+            afterEach {
+                // Don't worry about force_try in tests.
+                // swiftlint:disable:next force_try
+                try! inMemoryRealm.write {
+                    inMemoryRealm.deleteAll()
+                }
+                inMemoryRealm.invalidate()
+            }
+            context("store a few tags") {
```

Comment:
Can you also test the deleting tags as well?

Reply:
Addressed in recent commit